gemfiles/rails_5_beta4.gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
```

Comment:
Could you rename this to `gemfiles/Gemfile-rails.5.0.0.beta4`. This is my taste 😅